### PR TITLE
Refactor separate CommandHandler classes in ado2gh

### PR DESCRIPTION
**Pull request by dylan-smith**
Split out a CommandHandler class for all the ado2gh commands. This is in preparation for the next step in the refactor where we will change how we wire up the command/handlers using some new binding/reflection code.

There will be more PR's to follow with the rest of the commands.

Issue #638 

- [x] Did you write/update appropriate tests
- [x] Release notes updated (if appropriate)
- [x] Appropriate logging output
- [x] Issue linked
- [x] Docs updated (or issue created)

### CODE REVIEW

File: src/Octoshift/Octoshift.csproj [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<Project Sdk="Microsoft.NET.Sdk">
```
[the remark this answers is not in the document]
Comment (ArinGhazarian) [reply src/Octoshift/Octoshift.csproj:1]: we probably don't need this right ?

[the remark this answers is not in the document]
Comment (ArinGhazarian) [reply src/Octoshift/Octoshift.csproj:1]: Oh I see, nvm, it's weird 😕

File: src/ado2gh/Commands/AddTeamToRepoCommandHandler.cs [hunk 2]
```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Threading.Tasks;
+
+namespace OctoshiftCLI.AdoToGithub.Commands
```
Comment (ArinGhazarian) [inline src/ado2gh/Commands/AddTeamToRepoCommandHandler.cs:4]: wouldn't it be better to put the handlers in a separate directory? e.g. `OctoshiftCLI.AdoToGithub.Handlers`

Comment (dylan-smith) [reply src/ado2gh/Commands/AddTeamToRepoCommandHandler.cs:4]: agreed, I'll do that

File: src/ado2gh/Handlers/AddTeamToRepoCommandHandler.cs [hunk 3]
```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Threading.Tasks;
+using OctoshiftCLI.AdoToGithub.Commands;
+
+namespace OctoshiftCLI.AdoToGithub.Handlers
+{
```
Comment (ArinGhazarian) [inline src/ado2gh/Handlers/AddTeamToRepoCommandHandler.cs:6]: use file scoped namespaces to reduce indentation.

File: src/ado2gh/Handlers/ConfigureAutoLinkCommandHandler.cs [hunk 4]
```diff
@@ -0,0 +1,63 @@
+﻿using System;
+using System.Linq;
+using System.Threading.Tasks;
+using OctoshiftCLI.AdoToGithub.Commands;
+
+namespace OctoshiftCLI.AdoToGithub.Handlers
+{
```
Comment (ArinGhazarian) [inline src/ado2gh/Handlers/ConfigureAutoLinkCommandHandler.cs:7]: use file scoped namespaces to reduce indentation.

File: src/ado2gh/Handlers/DisableRepoCommandHandler.cs [hunk 5]
```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.Linq;
+using System.Threading.Tasks;
+using OctoshiftCLI.AdoToGithub.Commands;
+
+namespace OctoshiftCLI.AdoToGithub.Handlers
+{
```
Comment (ArinGhazarian) [inline src/ado2gh/Handlers/DisableRepoCommandHandler.cs:7]: use file scoped namespaces to reduce indentation.

File: src/ado2gh/Handlers/GenerateScriptCommandHandler.cs [hunk 6]
```diff
@@ -0,0 +1,529 @@
...
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using OctoshiftCLI.AdoToGithub.Commands;
+using OctoshiftCLI.Contracts;
+using OctoshiftCLI.Extensions;
+
+namespace OctoshiftCLI.AdoToGithub.Handlers
+{
```
Comment (ArinGhazarian) [inline src/ado2gh/Handlers/GenerateScriptCommandHandler.cs:12]: use file scoped namespaces to reduce indentation.

File: src/ado2gh/Handlers/IntegrateBoardsCommandHandler.cs [hunk 7]
```diff
@@ -0,0 +1,81 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using OctoshiftCLI.AdoToGithub.Commands;
+
+namespace OctoshiftCLI.AdoToGithub.Handlers
+{
```
Comment (ArinGhazarian) [inline src/ado2gh/Handlers/IntegrateBoardsCommandHandler.cs:8]: use file scoped namespaces to reduce indentation.

File: src/ado2gh/Handlers/InventoryReportCommandHandler.cs [hunk 8]
```diff
@@ -0,0 +1,107 @@
...
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using OctoshiftCLI.AdoToGithub.Commands;
+using OctoshiftCLI.Extensions;
+
+namespace OctoshiftCLI.AdoToGithub.Handlers
+{
+    public class InventoryReportCommandHandler
+    {
```
Comment (ArinGhazarian) [inline src/ado2gh/Handlers/InventoryReportCommandHandler.cs:11]: use file scoped namespaces to reduce indentation.

File: src/ado2gh/Handlers/LockRepoCommandHandler.cs [hunk 9]
```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Threading.Tasks;
+using OctoshiftCLI.AdoToGithub.Commands;
+
+namespace OctoshiftCLI.AdoToGithub.Handlers
+{
```
Comment (ArinGhazarian) [inline src/ado2gh/Handlers/LockRepoCommandHandler.cs:6]: use file scoped namespaces to reduce indentation.

File: src/ado2gh/Handlers/MigrateRepoCommandHandler.cs [hunk 10]
```diff
@@ -0,0 +1,100 @@
+﻿using System;
+using System.Threading.Tasks;
+using OctoshiftCLI.AdoToGithub.Commands;
+
+namespace OctoshiftCLI.AdoToGithub.Handlers
+{
```
Comment (ArinGhazarian) [inline src/ado2gh/Handlers/MigrateRepoCommandHandler.cs:6]: use file scoped namespaces to reduce indentation.

File: src/ado2gh/Handlers/RewirePipelineCommandHandler.cs [hunk 11]
```diff
@@ -0,0 +1,48 @@
+﻿using System;
+using System.Threading.Tasks;
+using OctoshiftCLI.AdoToGithub.Commands;
+
+namespace OctoshiftCLI.AdoToGithub.Handlers
+{
+    public class RewirePipelineCommandHandler
+    {
```
Comment (ArinGhazarian) [inline src/ado2gh/Handlers/RewirePipelineCommandHandler.cs:8]: use file scoped namespaces to reduce indentation.

File: src/ado2gh/Handlers/ShareServiceConnectionCommandHandler.cs [hunk 12]
```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.Threading.Tasks;
+using OctoshiftCLI.AdoToGithub.Commands;
+
+namespace OctoshiftCLI.AdoToGithub.Handlers
+{
```
Comment (ArinGhazarian) [inline src/ado2gh/Handlers/ShareServiceConnectionCommandHandler.cs:6]: use file scoped namespaces to reduce indentation.

**Review by ArinGhazarian — CHANGES_REQUESTED** [review]
Looks great but we need to use file scoped namespaces for the new files (at some point we will also need to apply it to older files too).

File: src/ado2gh/Handlers/AddTeamToRepoCommandHandler.cs (hunk 3 above)
Comment (dylan-smith) [reply src/ado2gh/Handlers/AddTeamToRepoCommandHandler.cs:6]: good catch, I'll fix them all up

Comment (dylan-smith) [reply src/ado2gh/Handlers/AddTeamToRepoCommandHandler.cs:6]: I wonder if there's an .editorconfig setting we can turn on for this so linters catch it

**Review by ArinGhazarian — APPROVED** [review]
🥇

**Pull request merged**